cli/src/local/watcher.js

```diff
@@ -125,11 +135,37 @@ class LocalWatcher {
     })
   }
 
+  handleEvents (events: ChokidarFSEvent[]) {
+    log.debug(`Flushed ${events.length} events`)
+    for (let e of events) {
+      switch (e.type) {
+        case 'add':
+          this.onAddFile(e.path, e.stats)
+          break
+        case 'addDir':
+          this.onAddDir(e.path, e.stats)
+          break
+        case 'change':
+          this.onChange(e.path, e.stats)
+          break
+        case 'unlink':
+          this.onUnlinkFile(e.path)
+          break
+        case 'unlinkDir':
+          this.onUnlinkDir(e.path)
+          break
+        default:
+          log.error(`Unknown event type: ${JSON.stringify(e.type)}`)
+      }
+    }
+  }
+
   stop () {
     if (this.watcher) {
       this.watcher.close()
       this.watcher = null
     }
+    this.buffer.switchMode('idle')
```

Comment:
Shouldn't we flush there ?

Reply:
The buffer flushes automatically when switching mode, because in both cases we want to flush anyway.